vue/components/ui/molecules/select-checkboxes/select-checkboxes.vue

```diff
@@ -152,6 +221,21 @@ export const SelectCheckboxes = {
                 }))
             ];
         },
+        _filterCheckboxes() {
+            if (this.filterCheckboxes) return this.filterCheckboxes;
+            return (items, filter) => {
+                return items.filter(item => {
+                    const itemValue = `${item.label || item.value}`.toLowerCase();

```

Comment:
```suggestion
                    const itemValue = (item.label || item.value).toLowerCase();
```
Don't think we need to convert this to a string

Reply:
Value could be something that isn't a string, it's safer doing this way

Reply:
Can it be a Number too? Because if it can only a String we wouldn't need. In case it is a null or undefined we shouldn't even be converting I believe.
But the `${}` surely "solves" all of it